test/ghtype_tests.jl

```diff
@@ -543,12 +543,6 @@ hook_json = JSON.parse("""
     "name": "web",
     "events": ["push", "pull_request"],
     "active": true,
-    "config": {
-      "content_type": "json",
-      "secret": "********",
-      "url": "http://webhook.com:1234/",
-      "insecure_ssl": "0"
-    },
```

Comment:
Why remove this? If this should pass, and it was causing v0.4 errors, we should probably fix the errors (or drop v0.4 support) rather than remove the failing part of the test.